src/Firebase/Auth/ApiClient.php

```diff
@@ -74,13 +74,13 @@ public function updateUser(Request\UpdateUser $request): ResponseInterface
     }
 
     /**
+     * @throws AuthException
```

Comment:
Could you please revert the changes in the PHPDocs except for the new methods? 🙈 (I make this emoji because, on the one hand, I know that this is just a minor/non-relevant thing, but on the other hand, it's important to me 😅)